hack/make-rules/test-cmd-util.sh

```diff
@@ -5195,7 +5195,8 @@ runTests() {
     kube::test::get_object_assert 'clusterrolebindings -l test-cmd=auth' "{{range.items}}{{$id_field}}:{{end}}" 'testing-CRB:'
     kube::test::get_object_assert 'clusterroles -l test-cmd=auth' "{{range.items}}{{$id_field}}:{{end}}" 'testing-CR:'
 
-    kubectl delete "${kube_flags[@]}" rolebindings,role,clusterroles,clusterrolebindings -n some-other-random -l test-cmd=auth
```

Comment:
I actually think uses like this are exactly the reason we shouldn't make this breaking change

Reply:
I don't feel too strong in any direction, to be honest.